src/Network/NWPath.cs

```diff
@@ -14,7 +14,9 @@
 using CoreFoundation;
 
 namespace Network {
-	public enum NWPathStatus {
+
+	[TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+	public enum NWPathStatus : long {
```

Comment:
`: long` is not correct for Network.framework

It's defined as
```
typedef enum {
	/*! @const nw_path_status_invalid The path is not valid */
	nw_path_status_invalid = 0,
	/*! @const nw_path_status_satisfied The path has a usable route upon which to send and receive data */
	nw_path_status_satisfied = 1,
	/*! @const nw_path_status_unsatisfied The path does not have a usable route. This may be due to a network interface being down, or due to system policy.  */
	nw_path_status_unsatisfied = 2,
	/*! @const nw_path_status_satisfiable The path does not currently have a usable route, but a connection attempt will trigger network attachment */
	nw_path_status_satisfiable = 3,
} nw_path_status_t;
```

Reply:
since it's confusing let's add a comment like:
```
// this maps to `nw_path_status_t` in Network/Headers/path.h (and not the enum from NetworkExtension)
```